packages/utils/src/ColorUtils.ts

```diff
@@ -4,25 +4,30 @@ class ColorUtils {
    *
    * @param {string} background the background color
    */
-  static isDark(background) {
+  static isDark(background: string): boolean {
     const d = document.createElement('div');
     d.style.display = 'none';
     d.style.color = background;
-    const color = getComputedStyle(document.body.appendChild(d))
-      .color.match(/\d+/g)
-      .map(a => parseInt(a, 10));
+    const colorTokens = getComputedStyle(
+      document.body.appendChild(d)
+    ).color.match(/\d+/g);
+    let color: number[] = [];
+    if (colorTokens) {
+      color = colorTokens.map(a => parseInt(a, 10));
+    } else {
+      throw new Error(
+        'Invalid color received. Expected something like ["123", "123", "123"].'
+      );
+    }
     document.body.removeChild(d);
     const brightness = ColorUtils.getBrightness(color);
     return brightness < 125;
   }
 
-  static getBrightness(color) {
+  static getBrightness(color: number[]): number {
```

Comment:
You can change the type to `[number, number, number]` to require it be an array of 3 numbers passed in.

However, this would mean you need to do some assertion before calling this method typically which would break it if we used anywhere else (like the call in `isDark`).

Could just add an array length check and error if it's not 3. Or leave it as is. We don't seem to call this anywhere in OSS except for in `isDark` above